tests/lib/rules/require-prop-comment.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * @author CZB
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/require-prop-comment')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('require-prop-comment', rule, {
+  valid: [
+    {
+      code: `
+      <script>
+      import { defineComponent } from '@vue/composition-api'
+
+      export default defineComponent({
+        props: {
+          /**
+           * a comment
+           */
+          a: Number,
+        }
+      })
+      </script>
+      `
+    },
+    {
+      code: `
+      <script setup>
+      const props = defineProps({
+        /*
+         * a comment
+         */
+        a: Number,
+        /**
+         * b comment
+         */
+        b: Number
+      })
+      </script>
+      `,
+      options: [{ type: 'block' }]
+    },
+    {
+      code: `
+      import { defineComponent } from '@vue/composition-api'
+      export const ComponentName = defineComponent({
+        props: {
+          // a comment
+          // a other comment
+          a: Number
+        }
+      })
+      `,
+      options: [{ type: 'line' }]
+    },
+    {
+      code: `
+      import { defineComponent } from '@vue/composition-api'
+      export const ComponentName = defineComponent({
+        props: {
+          /**
+           * a comment
+           */
+          a: Number,
+          // a comment
+          b: Number
+        }
+      })
+      `,
+      options: [{ type: 'any' }]
+    }
+  ],
+  invalid: [
+    {
+      code: `
+      <script>
+      import { defineComponent } from '@vue/composition-api'
+
+      export default defineComponent({
+        props: {
+          a: Number,
+          // c comment
+          c: Number
+        }
+      })
+      </script>
+      `,
+      options: [{ type: 'block' }],
+      errors: [
+        {
+          line: 7,
+          message: 'The "a" property should have a block comment.'
+        },
```

Comment:
Please add the `column` to all expected errors.

Reply:
ok